orangecontrib/text/widgets/owwordcloud.py

```diff
@@ -20,6 +20,7 @@ class IO:
     CORPUS = 'Corpus'
     TOPIC = 'Topic'
     SELECTED_WORDS = 'Selected Words'
+    WEIGHTS = 'Weights'
```

Comment:
Should we rather use `Word Counts` instead of `Weights`? AFAIK we only use this for Corpus and not also for Topics, right? If so we always output counts. Or at least `Words & Weights`?